phpstan.neon

```diff
@@ -2,23 +2,25 @@ includes:
     - phar://phpstan.phar/conf/bleedingEdge.neon
     - vendor/szepeviktor/phpstan-wordpress/extension.neon
 parameters:
-  checkGenericClassInNonGenericObjectType: false
-  checkMissingIterableValueType: false
-
-  ignoreErrors:
-    - '#Unsafe usage of new static\(\).#'
-    - '#Call to static method [a-zA-Z_\(\)]+ on an unknown class Akismet.#'
-    - '#Call to protected method get_views\(\) of class WP_Posts_List_Table.#'
-    - '#Function pll_[a-z_]+ not found.#'
-    - '#Constant DB_|ICL_|WP[a-zA-Z_]+ not found.#'
-    - '#^Function apply_filters(_ref_array)? invoked with [34567] parameters, 2 required\.$#'
-    - '#Access to an undefined property WP_|GeminiLabs\\SiteReviews\\(Arguments|Request)::(.+)#'
-    - '/^Parameter #2 \$callback of function array_filter expects callable\(mixed, mixed\): bool, \S+ given\.$/'
   level: 5
-  paths:
-    - plugin
-  scanFiles:
+  bootstrapFiles:
+    - tests/phpstan/bootstrap.php
     - autoload.php
     - helpers.php
-  scanDirectories:
-    - vendors
+  paths:
+    - plugin/
+  ignoreErrors:
+    # Uses func_get_args()
+    - '#^Function apply_filters(_ref_array)? invoked with [34567] parameters, 2 required\.$#'
+    # TODO
+    - '#^Unsafe usage of new static\(\)\.$#'
+    # TODO Add Akismet stubs
```

Comment:
Would you be able to provide example stubs for this?

Reply:
No, but if you take Akismet's relevant file  that contains the public API them you can generate it yourself https://github.com/GiacoCorsiglia/php-stubs-generator
One command only :)